### PR TITLE
TlBank

### PR DESCRIPTION
**Pull request by LordRanchoatos**
Superseded by #195 
TlBank integration

### CODE REVIEW

**Review by MantisClone — CHANGES_REQUESTED** [review]
Please upgrade to Node 16 or above.

Vercel errors:

```
Running build in Washington, D.C., USA (East) – iad1
--
05:20:49.876 | Cloning github.com/BanklessDAO/bankless-website (Branch: main, Commit: cfef785)
05:20:56.801 | Cloning completed: 6.925s
05:20:57.119 | Restored build cache
05:20:57.161 | Running "vercel build"
05:20:57.670 | Vercel CLI 31.0.1
05:20:57.983 | Error: Node.js version 14.x has reached End-of-Life. Deployments created on or after 2023-08-15 will fail to build. Please set Node.js Version to 18.x in your Project Settings to use Node.js 18.
05:20:57.991 | Installing dependencies...
05:20:58.293 | yarn install v1.22.17
05:20:58.388 | [1/4] Resolving packages...
05:20:58.988 | [2/4] Fetching packages...
05:21:33.934 | error svelte-i18n@3.6.0: The engine "node" is incompatible with this module. Expected version ">= 16". Got "14.21.3"
05:21:33.935 | error Found incompatible module.
05:21:33.935 | info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
05:21:33.969 | Error: Command "yarn install" exited with 1
```

**Review by MantisClone — CHANGES_REQUESTED** [review]
There are a significant number of changes here. Can you please add a PR description or link to documentation that describes this code's design and intended operation? Such a description will drastically shorten the time it takes me to review. (It will take me longer if I figure it out myself). I think a discussion of why core design decisions were made would be helpful too.

File: tlUtils/config.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+export const MAINNET_RPC_URL = `https://mainnet.infura.io/v3/9344a875cace468ca6c8029147cda56a`
+export const GOERLI_RPC_URL = `https://goerli.infura.io/v3/9344a875cace468ca6c8029147cda56a`
```
Comment (MantisClone) [inline tlUtils/config.js:2]: I think we'll need to create repository or vercel secrets for these Project IDs.

File: tlUtils/config.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import frontierModule from '@web3-onboard/frontier'
+import coinbaseWalletModule from '@web3-onboard/coinbase'
+import walletConnectModule from '@web3-onboard/walletconnect'
+
+import injectedModule from '@web3-onboard/injected-wallets'
+const injected = injectedModule()
+const coinbaseWalletSdk = coinbaseWalletModule()
+const walletConnect = walletConnectModule({
+  version: 2,
+  projectId: 'e1726502f108ccc7845fc598e1a605f3',
```
Comment (MantisClone) [inline tlUtils/config.js:28]: I think we'll need to create a repository or vercel secret for this project ID.

**Review by MantisClone — CHANGES_REQUESTED** [review]

**Comment by MantisClone** [comment]
CC @Icedcool

File: pages/tlBank.tsx [hunk 3]
```diff
@@ -0,0 +1,825 @@
...
+  MenuList,
+  MenuItem,
+  Spinner,
+} from '@chakra-ui/react'
+import { init, useConnectWallet } from '@web3-onboard/react'
+import { useSetChain } from '@web3-onboard/react'
+import { BigNumber, ethers } from 'ethers'
+import { providers } from '@0xsequence/multicall'
+
+const API = '9176eee3-12fa-431c-93c5-27d1f40d4c91'
```
[the remark this answers is not in the document]
Comment (Icedcool) [reply pages/tlBank.tsx:54]: A repository works for me?

I would defer to what you think is best.

**Comment by Icedcool** [comment]
So I just forked https://github.com/LordRanchoatos/bankless-website and built it locally, and it is saying the node version is "17.0.14".

**Comment by MantisClone** [comment]
> Please upgrade to Node 16 or above.

The node version was a Vercel setting set to version 14! I fixed the Vercel build by upgrading to Node 18. CC @Icedcool @LordRanchoatos 
image

**Comment by MantisClone** [comment]
@Icedcool I quickly clicked through the top-nav bar but couldn't find the TLBank page. At what URL path is it located?

**Comment by Icedcool** [comment]
@MantisClone Should just be */tlBank.

**Comment by Icedcool** [comment]
https://bankless-website-git-fork-lordranchoatos-main-banklessdao.vercel.app/tlBank

Looking good!

**Pull request closed without merging**